Description: disable test_security_curve
 It hangs in buildds that not x86/x64 based.
Author: Laszlo Boszormenyi (GCS) <gcs@debian.org>
Forwarded: no
Last-Update: 2014-10-23

---

--- zeromq3-4.0.5+dfsg.orig/tests/Makefile.am
+++ zeromq3-4.0.5+dfsg/tests/Makefile.am
@@ -27,7 +27,6 @@ noinst_PROGRAMS = test_system \
                   test_ctx_destroy \
                   test_security_null \
                   test_security_plain \
-                  test_security_curve \
                   test_iov \
                   test_spec_req \
                   test_spec_rep \
